Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:22-bullseye as node_builder
 WORKDIR /app
 COPY . .
 
-RUN npm install && npm run sass
+RUN npm install
```

Comment:
Do you not need to run the new `npm run dependencies` command for this to be equivalent?

Reply:
Ah, nevermind, I see this is automatically run 👍🏻

Reply:
dependencies is automatically triggered after npm makes any changes to the node_modules folder, so after any install or most updates